build/pipelines/release.yml

```diff
@@ -343,6 +374,97 @@ jobs:
         PathtoPublish: $(System.ArtifactsDirectory)
         ArtifactName: appxbundle-signed-$(TerminalTargetWindowsVersion)
 
+- ${{ if eq(parameters.buildConPTY, true) }}:
+  - job: PackageAndSignConPTY
+    strategy:
+      matrix:
+        ${{ each config in parameters.buildConfigurations }}:
+          ${{ config }}:
+            BuildConfiguration: ${{ config }}
+    displayName: Create NuGet Package (ConPTY)
+    dependsOn: Build
+    steps:
+    - checkout: self
+      clean: true
+      fetchDepth: 1
+      submodules: true
+      persistCredentials: True
+    - task: PkgESSetupBuild@12
+      displayName: Package ES - Setup Build
+      inputs:
+        disableOutputRedirect: true
+    - ${{ each platform in parameters.buildPlatforms }}:
+      - task: DownloadBuildArtifacts@0
+        displayName: Download ${{ platform }} ConPTY binaries
+        inputs:
+          artifactName: conpty-dll-${{ platform }}-$(BuildConfiguration)-Win10

```

Comment:
Why's it Win10 here and not `$(TerminalTargetWindowsVersion)`?

Reply:
this is a secret dependency in our build system. There is no need for us to produce a Win10 *and* Win11 version of the ConPTY or WPF packages, but they depend on build artifacts from the "main" versioned build.

It would be a waste of time to build both of them, so I made the decision to build them out of the Windows 10 copy of the "main" build. Does that seem reasonable?